system/include/emscripten/emscripten.h

```diff
@@ -271,7 +271,19 @@ emscripten_coroutine emscripten_coroutine_create(em_arg_callback_func func, void
 int emscripten_coroutine_next(emscripten_coroutine);
 void emscripten_yield(void);
 
-
+//  node.js : getenv
+#define getenv(name)\
```

Comment:
Strange. But we should figure this out - we cant define getenv in our header, it might break various things and is a potential source of confusion for developers.